include/deal.II/base/parameter_handler.h

```diff
@@ -1332,10 +1333,15 @@ class ParameterHandler : public Subscriptor
   void
   set(const std::string &entry_name, const bool new_value);
 
-
   /**
    * Print all parameters with the given style to <tt>out</tt>.
    *
+   * Before printing, all current parameters and subsections are printed sorted
+   * alphabetically by default.
+   * This option can be disabled setting the last parameter @p sort_alphabetical
+   * to <kbd>false</kbd>: in this case entries are printed in the same order
```

Comment:
what is <kbd>? Can you use ``@p false`` instead?

Reply:
It is an alias for the deprecated `<tt>`, but it should be rendered likewise.
However it is fixed now!

Reply:
Interesting, let's hope <tt> will not disappear any time soon, considering we are using it all over the place.